extensions/amp-story/1.0/amp-story.css

```diff
@@ -863,42 +863,33 @@ amp-story .amp-video-eq,
 .i-amphtml-story-page-play-icon {
   background-image: url('data:image/svg+xml;charset=utf-8,<svg xmlns="http://www.w3.org/2000/svg" width="24px" height="24px" viewBox="0 0 48 48" fill="#FFF"><path d="M0 0h48v48H0z" fill="none"/><path d="M24 4C12.95 4 4 12.95 4 24s8.95 20 20 20 20-8.95 20-20S35.05 4 24 4zm-4 29V15l12 9-12 9z"/></svg>') !important;
 }
-
-.i-amphtml-story-reaction {
-  align-self: center;
+.i-amphtml-story-reaction-component {
+  align-self: center !important;
```

Comment:
Please remove the !important, you're preventing publishers from overriding the position to align the quiz/poll on a corner. I assume that it's breaking this test https://github.com/ampproject/amphtml/blob/master/examples/visual-tests/amp-story/amp-story-reaction-quiz-sizing-positioning.html